packages/playground/src/explorer/components/node_details.vue

```diff
@@ -99,35 +114,56 @@ export default {
     PublicConfigDetailsCard,
   },
 
-  setup(props) {
+  setup(props, { emit }) {
     const loading = ref(false);
     const dialog = ref(false);
+    const isError = ref(false);
+    const errorMessage = ref("");
     const route = useRoute();
     const node = ref<GridNode>(nodeInitializer);
 
-    watch(
-      () => props.nodeId,
-      async (nodeId: number) => {
-        if (nodeId > 0) {
+    function closeDialog(newValue: boolean) {
+      isError.value = false;
```

Comment:
this make the card appear for a while. i think it is better to reset the value before the request